nncf/experimental/tensor/torch_functions.py

```diff
@@ -178,25 +178,165 @@ def _(x: torch.Tensor, axis: int = 0) -> List[torch.Tensor]:
+    return torch.inf
+
+
+def _expand_scalars_in_tensor_list(tensor_list: List[torch.Tensor]) -> List[torch.Tensor]:
+    retval = []
+    for t in tensor_list:
+        if len(t.size()) == 0:
+            retval.append(t.reshape(1))
+        else:
+            retval.append(t)
+    return retval
+
+
+@fns.concatenate.register(torch.Tensor)
+def _(x: List[torch.Tensor], axis: int = 0) -> torch.Tensor:
+    expanded_scalar_list = _expand_scalars_in_tensor_list(x)
+    return torch.concatenate(expanded_scalar_list, dim=axis)
+
+
+@fns.min_of_list.register(torch.Tensor)
+def _(x: List[torch.Tensor], axis: int = 0) -> torch.Tensor:
+    cated = torch.concatenate(_expand_scalars_in_tensor_list(x))
+    return torch.min(cated, dim=axis).values
+
+
+@fns.max_of_list.register(torch.Tensor)
+def _(x: List[torch.Tensor], axis: int = 0) -> torch.Tensor:
+    cated = torch.concatenate(_expand_scalars_in_tensor_list(x))
+    return torch.max(cated, dim=axis).values
+
+
+@fns.amax.register(torch.Tensor)
+def _(a: torch.Tensor, axis: Optional[List[int]] = None, keepdims: Optional[bool] = None) -> torch.Tensor:
+    if keepdims is None:
+        keepdims = False
+    return torch.amax(a, dim=axis, keepdim=keepdims)
+
+
+@fns.amin.register(torch.Tensor)
+def _(a: torch.Tensor, axis: Optional[List[int]] = None, keepdims: Optional[bool] = None) -> torch.Tensor:
+    if keepdims is None:
+        keepdims = False
+    return torch.amin(a, dim=axis, keepdim=keepdims)
+
+
+@fns.clip.register(torch.Tensor)
+def _(a: torch.Tensor, min_val: float, max_val: Optional[float] = None) -> torch.Tensor:
+    return torch.clip(a, min=min_val, max=max_val)
+
+
+@fns.sum.register(torch.Tensor)
+def _(a: torch.Tensor, axes: List[int]) -> torch.Tensor:
+    return torch.sum(a, dim=axes)
+
+
+@fns.transpose.register(torch.Tensor)
+def _(a: torch.Tensor, axes: List[int]) -> torch.Tensor:
+    return torch.permute(a, dims=axes)
+
+
+@fns.eps.register(torch.Tensor)
+def _(a: torch.Tensor, dtype: TensorDataType) -> float:
+    return torch.finfo(DTYPE_MAP[dtype]).eps
+
+
+@fns.median.register(torch.Tensor)
+def _(a: torch.Tensor, axis: Union[int, Tuple[int]] = None, keepdims: Optional[bool] = None) -> torch.Tensor:
+    return torch.quantile(a, q=0.5, dim=axis, keepdim=keepdims)
+
+
+@fns.power.register(torch.Tensor)
+def _(a: torch.Tensor, pwr: float) -> torch.Tensor:
+    return torch.pow(a, exponent=pwr)
+
+
+@fns.matmul.register(torch.Tensor)
+def _(
+    a: torch.Tensor,
+    b: torch.Tensor,
+) -> Union[np.ndarray, np.number]:
+    return torch.matmul(a, b)
+
+
+@fns.quantile.register(torch.Tensor)
+def _(
+    a: torch.Tensor,
+    q: Union[float, List[float]],
+    axis: Union[int, List[int]] = None,
+    keepdims: Optional[bool] = None,
+) -> Union[float, torch.Tensor]:
+    if not isinstance(axis, list):
+        return torch.quantile(a, q=q, dim=axis, keepdim=keepdims)
+    if len(axis) == 1:
+        return torch.quantile(a, q=q, dim=axis[0], keepdim=keepdims)
```

Comment:
`torch.quantile` does not work with big tensors as well
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/tensor_statistics/collectors.py#L168-L189